Taskfile.yml

```diff
@@ -943,7 +979,17 @@ tasks:
       # This below is added to test if ASO works fine in namespaces other than 'azureserviceoperator-system"
```

Comment:
I thought we'd removed this option - do we need this?

Reply:
No, I think we talked about it but then realized it actually worked and so instead of removing it we added a test to ensure that it works.